graphpype/interfaces/radatools/tests/test_rada.py

```diff
@@ -0,0 +1,36 @@
+"""test rada"""
+import os
+
+from graphpype.interfaces.radatools.rada import CommRada
+
+try:
+    import neuropycon_data as nd
+
+except ImportError:
+    print("Error, could not find neuropycon_data")
+
+
+data_path = os.path.join(nd.__path__[0], "data", "data_con")
+# conmat_file = os.path.join(data_path, "Z_cor_mat_resid_ts.npy")
+# coords_file = os.path.join(data_path, "ROI_MNI_coords-Atlas.txt")
+# Z_list_file = os.path.join(data_path, "data_graph", "Z_List.txt")
+Pajek_net_file = os.path.join(data_path, "data_graph", "Z_List.net")
+
+
+def test_CommRada():
```

Comment:
function names should not be in camel case. See pep8

Reply:
How about testing class? Because all nodes in nipype follows the convention of capital letters for class names [see this example](https://github.com/nipy/nipype/blob/50862c18d5bb469a3c9fbcb80ab79355c9b09e3f/nipype/interfaces/freesurfer/model.py)

Reply:
yes classes are in camel case according to pep8 but not functions.

Reply:
as far as I can tell, in nipype they use test_Class_Name (see [for example](https://github.com/nipy/nipype/blob/50862c18d5bb469a3c9fbcb80ab79355c9b09e3f/nipype/interfaces/freesurfer/tests/test_auto_ReconAll.py)), although it seems this example is automatically generated